nbconvert/exporters/export.py

```diff
@@ -31,9 +32,14 @@
 #-----------------------------------------------------------------------------
 
 def DocDecorator(f):
-    
+    """
+    Deprecated since version 5.0.
```

Comment:
(The upside of having deliberately curated API docs rather than generating them :-)